### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by RebelliousTT** [facebookresearch/TimeSformer#110]
I try to train the model you provide on ssv1, the loss decreases very slow. I am not sure if I have the right learning_rate(0.001) from the beginning. Do I have to pre-train it on Imagenet? If you can give me some advice on how to pre-train it, that would be helpful.
Sorry for taking your so much time. Thank you in advance

**Comment by chrisx599** [comment]
i meet the same problem, have u solved this?
